experiment/kind-nftables-e2e.sh

```diff
@@ -107,7 +107,7 @@ create_cluster() {
   fi
 
   # JSON map injected into featureGates config
-  feature_gates="{"NFTablesProxyMode":true}"
+  feature_gates='{"NFTablesProxyMode":"true"}'
```

Comment:
pretty sure you want

```suggestion
  feature_gates='{"NFTablesProxyMode":true}'
```

Reply:
👍